scripts/install2

```diff
@@ -322,6 +326,7 @@ fi
 if [ "$(checkModule 'WITH_YGGDRASIL_IPTUNNEL')" ]; then
     source yggdrasil-iptunnel/install
 fi
+source shared/hostname/install
```

Comment:
This script modifies `/opt/tomesh/nodeinfo.json`, but the nodeinfo module is installed below. It should be installed before?